front/src/modules/people/components/PeopleCard.tsx

```diff
@@ -51,14 +63,94 @@ const StyledJobTitle = styled.div`
     background: ${({ theme }) => theme.background.tertiary};
   }
 `;
+const StyledIconButton = styled(IconButton)`
+  &:hover {
```

Comment:
let's not override IconButtons or Buttons, they should have the correct styling already